medium_test.go

```diff
@@ -36,9 +36,18 @@ type apiTest struct {
 var m = NewClient("clientId", "clientSecret")
 
 var apiTests = []apiTest{
-	{"token", m.GetUser, nil,
+	{"token", m.GetUser, "",
 		"GET", "/v1/me", "application/json",
 		"null"},
+	{"token", m.GetUser, "@dummyUser",
```

Comment:
just a note that in practice this will be an opaque id (like "5303d74c64f66366f00cb9b2a94f3251bf5") rather than the @-username. This is fine for the unit test though.